syne_tune/optimizer/schedulers/searchers/kde/kde_searcher.py

```diff
@@ -71,40 +68,21 @@ class KernelDensityEstimator(StochasticAndFilterDuplicatesSearcher):
     def __init__(
         self,
         config_space: Dict[str, Any],
-        metric: str,
         points_to_evaluate: Optional[List[dict]] = None,
-        allow_duplicates: Optional[bool] = None,
-        mode: Optional[str] = None,
         num_min_data_points: Optional[int] = None,
-        top_n_percent: Optional[int] = None,
-        min_bandwidth: Optional[float] = None,
-        num_candidates: Optional[int] = None,
-        bandwidth_factor: Optional[int] = None,
-        random_fraction: Optional[float] = None,
-        **kwargs,
+        top_n_percent: int = 15,
+        min_bandwidth: float = 1e-3,
+        num_candidates: int = 64,
+        bandwidth_factor: int = 3,
+        random_fraction: float = 0.33,
+        random_seed: int = None,
```

Comment:
The right type would be (equivalent to Optional[int]):
```suggestion
        random_seed: int | None = None,
```